sql/2020/09_Performance/lighthouse_performace_audits.sql

```diff
@@ -0,0 +1,41 @@
+#standardSQL
+# Get summary of all lighthouse scores for a category
+# Note scores, weightings, groups and descriptions may be off in mixed months when new versions of Lighthouse roles out
+
+CREATE TEMPORARY FUNCTION getAudits(report STRING, category STRING)
+RETURNS ARRAY<STRUCT<id STRING, weight INT64, audit_group STRING, title STRING, description STRING, score INT64>> LANGUAGE js AS '''
+var $ = JSON.parse(report);
+var auditrefs = $.categories[category].auditRefs;
+var audits = $.audits;
+$ = null;
+var results = [];
+for (auditref of auditrefs) {
+  results.push({
+    id: auditref.id,
+    weight: auditref.weight,
+    audit_group: auditref.group,
+    description: audits[auditref.id].description,
+    score: audits[auditref.id].score
+  });
+}
+return results;
+''';
+
+SELECT
+  audits.id AS id,
+  COUNTIF(audits.score > 0) AS num_pages,
+  COUNT(0) AS total,
+  COUNTIF(audits.score > 0) / COUNT(0) AS pct,
```

Comment:
@obto pointed out in #1338 that not all audits are applicable for all sites and suggested better logic. I don't think that's the case for Performance as they all will be applicable if they exist, but let's use same logic anyway for consistency:

```suggestion
  COUNT(0) AS total,
  COUNTIF(audits.score IS NOT NULL) AS total_applicable,
  SAFE_DIVIDE(COUNTIF(audits.score > 0), COUNTIF(audits.score IS NOT NULL)) AS pct,
```

Reply:
BTW in case you're wondering, I keep the `total` column in anyways as it helps weed out some odd results - for example we still have some LH5 results in September (agents that have not upgraded yet for whatever reason I guess)

Reply:
Yes, makes sense. Yeah, I'd also keep it for transparency.